src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -369,6 +369,7 @@ default String getEffectiveAvatarUrl()
         BUG_HUNTER_LEVEL_2(14, "Bug Hunter Level 2"),
         VERIFIED_BOT(      16, "Verified Bot"),
         VERIFIED_DEVELOPER(17, "Early Verified Bot Developer"),
+        CERTIFIED_MODERATOR(18, "Discord Certified Moderator"),
```

Comment:
You should align the other entries accordingly, ro make it more easy to look at.